backends/dpdk/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ set(P4C_DPDK_SOURCES
     dpdkArch.cpp
     dpdkContext.cpp
     dpdkAsmOpt.cpp
+    dpdkPnaDirection.cpp
```

Comment:
dpdkMetadata would be better I feel. We might need to add similar stuff for _pass_, _loopedback_ metadata as well.